ibm/service/kubernetes/resource_ibm_container_worker_pool.go

```diff
@@ -42,7 +44,7 @@ func ResourceIBMContainerWorkerPool() *schema.Resource {
 					"cluster"),
 			},
 
-			"machine_type": {
+			"flavor": {
```

Comment:
please don't convert this resource to vpc, there is already a vpc_worker_pool one